br/pkg/lightning/backend/tidb/tidb.go

```diff
@@ -212,15 +214,24 @@ func (b *targetInfoGetter) FetchRemoteTableModels(ctx context.Context, schemaNam
 		// shard_row_id/auto random is only available after tidb v4.0.0
 		// `show table next_row_id` is also not available before tidb v4.0.0
 		if serverInfo.ServerType != version.ServerTypeTiDB || serverInfo.ServerVersion.Major < 4 {
+			results = tables
 			return nil
 		}
 
+		failpoint.Inject(
+			"FetchRemoteTableModels_BeforeFetchTableAutoIDInfos",
+			func() {
+				fmt.Println("failpoint: FetchRemoteTableModels_BeforeFetchTableAutoIDInfos")
```

Comment:
we can use nil inject function when the inject verb is sleep, like `FailAfterWriteRows`

Reply:
Yes, I understand your point. I print that out to verify in the integration test script that my `drop table` is executed before this failpoint's hook is triggered. This makes it easier to debug. Do you think it's OK to keep the printing message here?